src/server-middleware/api/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const defu = require('defu');
+
+const app = express();
+app.use(express.json());
+
+let runtimeConfig;
+app.use((res, req, next) => {
+  if (!runtimeConfig) {
+    // Load Nuxt config once, at runtime
+    const nuxtConfig = require('../../../nuxt.config');
+    runtimeConfig = defu(nuxtConfig.privateRuntimeConfig, nuxtConfig.publicRuntimeConfig);
+  }
+  next();
+});
+
+const debugMemoryUsage = require('./debug/memory-usage');
+app.get('/debug/memory-usage', debugMemoryUsage);
+
+const jiraServiceDesk = require('./jira/service-desk');
+app.post('/jira/service-desk', (res, req, next) => jiraServiceDesk(runtimeConfig.jira)(res, req, next));
```

Comment:
SonarCloud is raising a warning on the param order here.

Additionally, in fact I believe that we don't need to pass 'next' at all either.

Reply:
I have addressed the security hotspot, which is about the use of `x-powered-by` header, and also removed `next` from the arguments.